includes/Core/Assets/Assets.php

```diff
@@ -288,7 +288,8 @@ private function add_amp_dev_mode_attributes( $assets ) {
 		add_filter(
 			'script_loader_tag',
 			function ( $tag, $handle ) use ( $assets ) {
-				if ( $this->context->is_amp() && isset( $assets[ $handle ] ) && $assets[ $handle ] instanceof Script ) {
+				// TODO: 'hoverintent-js' can be removed from here at some point, see https://github.com/ampproject/amp-wp/pull/3928.
+				if ( $this->context->is_amp() && ( isset( $assets[ $handle ] ) && $assets[ $handle ] instanceof Script || 'hoverintent-js' === $handle ) ) {
```

Comment:
Wasn't the need to manually check for `hoverintent-js` made obsolete by https://github.com/ampproject/amp-wp/pull/3928?

Reply:
Yes indeed (see above), however this was only in the quite recent 1.4.2 version. We'll keep this around for a bit more to have support for older AMP plugin versions. I'd be curious to see what the AMP plugin version distribution is :)